src/MCPClient/lib/clientScripts/move_or_merge.py

```diff
@@ -82,7 +99,7 @@ def main(src, dst):
 
 def call(jobs):
     for job in jobs:
-        with job.JobContext():
+        with job.JobContext(logger=logger):
```

Comment:
I know I know :D I like logging here because it's like a utility script, but really, I haven't found a pattern I'm happy with yet either. Like you too, I'm just happy to have a bit more logging here. Do you think the granularity is okay?

Reply:
I think it's a matter of taste, but a couple seem to verbose to me: you could infer if `src` is a file or a directory from the initial `info`s?

Reply:
True, I can remove those two for sure. Will make another commit.